data/json/vehicle_parts.json

```diff
@@ -694,6 +694,28 @@
         ]
     },{
         "type" : "vehicle_part",
+        "id" : "washing_machine",
+        "name" : "washing machine",
+        "symbol" : "H",
+        "color" : "light_blue",
+        "broken_symbol" : "#",
+        "broken_color" : "light_blue",
+        "damage_modifier" : 80,
+        "durability" : 100,
+        "epower" : -249,
+        "size" : 160,
+        "item" : "washing_machine",
+        "difficulty" : 3,
+        "location" : "center",
+        "flags" : ["CARGO", "OBSTACLE", "WASHING_MACHINE", "COVERED", "TOOL_WRENCH", "ENABLED_DRAINS_EPOWER"],
+        "breaks_into" : [
+            {"item": "steel_lump", "count": [8, 13]},
+            {"item": "steel_chunk", "count": [8, 13]},
+            {"item": "scrap", "count": [8, 13]},
+            {"item": "hose", "prob": 50}
+        ]
```

Comment:
Not necessary in this PR, but it's a good idea to add disassemble recipes to new things, so that they always have some use, even the duplicates.